__tests__/app.test.js

```diff
@@ -83,4 +83,41 @@ describe("app", () => {
       return request(app).get("/api/articlez").expect(404);
     });
   });
+  describe("api/articles/article:id", () => {
+    it("should return 200 status : GET responds with one article object with the correct article_id", () => {
+      return request(app)
+        .get("/api/articles/3")
+        .expect(200)
+        .then((articleResponse) => {
+          const article = articleResponse.body;
+          expect(article).toEqual({
```

Comment:
toEqual here makes for a test that might break later if we need to add more info to the object. Consider a more flexible matcher like toMatchObject or objectContaining